src/uct/api/uct.h

```diff
@@ -222,13 +222,15 @@ typedef struct uct_tl_resource_desc {
                                                        is called. */
 
         /* Event notification */
-#define UCT_IFACE_FLAG_EVENT_FD       UCS_BIT(46) /**< Event notification via file descriptor is supported */
+#define UCT_IFACE_FLAG_EVENT_SEND_COMP    UCS_BIT(46) /**< Event notification of send completion is supported */
+#define UCT_IFACE_FLAG_EVENT_RECV_AM      UCS_BIT(47) /**< Event notification of active message receive is supported */
+#define UCT_IFACE_FLAG_EVENT_RECV_SIG_AM  UCS_BIT(48) /**< Event notification of signaled active message is supported */
```

Comment:
SIG_AM indicates support for signaled active messages, AM indicates support for regular active messages.
For example, shared memory transport may support only SIG_AM, IB supports both, self supports niether.